packages/sync/src/class-defaults.php

```diff
@@ -158,6 +158,7 @@ class Defaults {
 		'site_user_type',
 		'site_vertical',
 		'jetpack_excluded_extensions',
+		'amp-options',
```

Comment:
Perhaps this should only be included if `function_exists( 'is_amp_endpoint' )`? Then it wouldn't sync if AMP is not active.

Reply:
Good call. I changed the approach and moved this into the AMP support file.